### PR TITLE
gen-nncp.sh - add IPv6 support

### PR DESCRIPTION
**Pull request by hjensas**
When NETWORK_ISOLATION_IPV6 (default: false) is true, configure the NNCP for IPv6. Likewize NETWORK_ISOLATION_IPV4 (default: true) controls if IPv4 is configured.

Enabling both IPv6 and IPv4 should work for dual stack.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1874,10 +1878,20 @@ nncp: export BGP_2_IP_ADDRESS=${NNCP_BGP_2_IP_ADDRESS}
...
 endif
+ifeq ($(NETWORK_ISOLATION_IPV6), true)
+nncp: export IPV6_ENABLED=true
+nncp: export CTLPLANE_IPV6_ADDRESS_PREFIX=${NNCP_CTLPLANE_IPV6_ADDRESS_PREFIX}
+nncp: export CTLPLANE_IPV6_ADDRESS_SUFFIX=${NNCP_CTLPLANE_IPV6_ADDRESS_SUFFIX}
+nncp: export GATEWAY_IPV6=${NNCP_GATEWAY_IPV6}
+nncp: export DNS_SERVER_IPV6=${NNCP_DNS_SERVER_IPV6}
+endif
+ifeq ($(NETWORK_ISOLATION_IPV4), true)
+nncp: export IPV4_ENABLED=true
```
[the remark this answers is not in the document]
Comment (hjensas) [reply Makefile:1891]: doh, forgot to set the defaults ... added:
```
NETWORK_ISOLATION_IPV4 ?= true
NETWORK_ISOLATION_IPV6 ?= false
```

**Comment by hjensas** [comment]
recheck

**Comment by hjensas** [comment]
recheck

**Comment by hjensas** [comment]
recheck

**Review by fao89 — COMMENTED** [review]
/approve

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**